noxfile.py

```diff
@@ -0,0 +1,124 @@
+import nox
+from nox_poetry import Session, session
+
+PYTHON_VERSIONS = ["3.11", "3.10", "3.9", "3.8", "3.7"]
+
+
+@session(python=PYTHON_VERSIONS, name="Tests", tags=["tests"])
+def tests(session: Session) -> None:
+    session.run_always("poetry", "install", external=True)
+
+    session.run(
+        "pytest",
+        "--cov=strawberry",
+        "--cov-append",
+        "--cov-report=xml",
+        "-n",
+        "auto",
+        "--showlocals",
+        "-vv",
+        "-m",
+        "not starlette",
+        "-m",
+        "not django",
+        "-m",
+        "not starlite",
+        "--ignore=tests/mypy",
+        "--ignore=tests/pyright",
+    )
+
+
+@session(python=["3.11"], name="Django tests", tags=["tests"])
+@nox.parametrize("django", ["4.2.0", "4.1.0", "4.0.0", "3.2.0"])
+def tests_django(session: Session, django: str) -> None:
+    session.run_always("poetry", "install", external=True)
+
+    session._session.install(f"django~={django}")  # type: ignore
+
+    session.run(
+        "pytest",
+        "--cov=strawberry",
+        "--cov-append",
+        "--cov-report=xml",
+        "-n",
+        "auto",
+        "--showlocals",
+        "-vv",
+        "-m",
+        "django",
+    )
+
+
+@session(python=["3.11"], name="Starlette tests", tags=["tests"])
+@nox.parametrize("starlette", ["0.28.0", "0.27.0", "0.26.1"])
+def tests_starlette(session: Session, starlette: str) -> None:
+    session.run_always("poetry", "install", external=True)
+
+    session._session.install(f"starlette=={starlette}")  # type: ignore
```

Comment:
Also use `~=` in here as well?

```suggestion
    session._session.install(f"starlette~={starlette}")  # type: ignore
```

Reply:
I think we want to be specific here since it’s still 0.x :)

Reply:
Oh, I see. Didn't notice that

So ignore this comment :)